service/websocket_test.go

```diff
@@ -0,0 +1,71 @@
+package service
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/sprawl/sprawl/errors"
+	"github.com/sprawl/sprawl/pb"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/gorilla/websocket"
+)
+
+var upgrader = websocket.Upgrader{}
+
+var testChannel *pb.Channel = &pb.Channel{Id: []byte("testChannel")}
+var testOrder *pb.Order = &pb.Order{Asset: string("ETH"), CounterAsset: string("BTC"), Amount: 52152, Price: 0.2, Id: []byte("jgkahgkjal")}
+var testOrderInBytes []byte
+var testWireMessage *pb.WireMessage
+
+func StartMockServer(websocketService *WebsocketService) (ws *websocket.Conn, s *httptest.Server, err error) {
```

Comment:
Is there a reason why you don't use the `websocketService.Start()` for this instead of mocking the server here? Are there some benefits I'm missing?